e2e/client/playwright/authoring.spec.ts

```diff
@@ -38,3 +38,28 @@ test('can apply "populate abstract" macro', async ({page}) => {
         page.locator(s('authoring', 'authoring-field=abstract')).getByRole('textbox'),
     ).toHaveText('test sport story body');
 });
+
+test('can embed if url is valid', async ({page}) => {
+    await restoreDatabaseSnapshot();
+
+    const monitoring = new Monitoring(page);
+
+    await page.goto('/#/workspace/monitoring');
+
+    await monitoring.selectDesk('Sports');
+
+    await page.locator(
+        s('monitoring-group=Sports / Working Stage', 'article-item=test sports story'),
+    ).dblclick();
+
+    await expect(
+        page.locator(s('authoring', 'authoring-field=body_html')).getByRole('textbox'),
+    ).toHaveText('test sport story body');
+
+    await page.locator(s('Embed')).getByRole('button').click();
```

Comment:
avoid global selectors like `s('embed')`. If someone modifies the content profile in the future and enables embeds in other fields it might crash your test. Since there may possible be multiple fields and you want to work only with body HTML, use `(s('authoring', 'authoring-field=body_html', 'embed'))`